src/components/MetricChart/types.ts

```diff
@@ -0,0 +1,32 @@
+export type Metric =
+    | 'queries.requests'
+    | 'queries.latencies.p50'
+    | 'queries.latencies.p75'
+    | 'queries.latencies.p90'
+    | 'queries.latencies.p99'
+    | 'resources.cpu.usage'
+    | 'resources.memory.used_bytes'
+    | 'resources.storage.used_bytes';
```

Comment:
Currently we have quite short list of metrics, so I enumerated them all here. Further, if more metrics will be added, some more complex types may be used: 
```
type Percentile = 'p50' | 'p75' | 'p90' | 'p99';
type QueryMetric = `queries.{string}.{Percentile}`;
type BytesMetric = `resources.{string}.used_bytes`
type Metric = QueryMetric | BytesMetric
```